java/junit-client-sample/src/test/java/com/smartbear/readyapi/SimpleTest.java

```diff
@@ -11,36 +11,35 @@
 import static com.smartbear.readyapi.client.teststeps.TestSteps.getRequest;
 import static org.junit.Assert.assertEquals;
 
-public class SimpleTest extends ApiTestBase
-{
+public class SimpleTest extends ApiTestBase {
     @Test
     public void simpleCountTest() throws Exception {
         TestRecipe recipe = newTestRecipe()
-            .addStep(
-                getRequest("https://api.swaggerhub.com/apis")
-                    .addQueryParameter("query", "testserver")
-                    .assertJsonContent("$.totalCount", "2" )
-            )
-            .buildTestRecipe();
+                .addStep(
+                        getRequest("https://api.swaggerhub.com/apis")
+                                .addQueryParameter("query", "testserver")
+                                .assertJsonContent("$.totalCount", "3")
```

Comment:
I know - but that's not necessarily a bad thing - since it shows what a failed test looks like